petab/visualize/data_overview.py

```diff
@@ -72,7 +79,7 @@ def get_data_per_observable(measurement_df: pd.DataFrame) -> pd.DataFrame:
     return data_per_observable
 
 
-def main():
+def main(outdir: Union[Path, str] = None):
     """Data overview generation with example data from the repository for
     testing
     """
```

Comment:
Why is this test case code here and not in a test script?

Reply:
:see_no_evil: